build/make_addon.py

```diff
@@ -0,0 +1,30 @@
+from zipfile import ZipFile, ZIP_DEFLATED
+from os import path, walk, listdir
+from datetime import date
+
+build_script_dir = path.dirname(path.abspath(__file__))
+today = date.today()
+filedate = today.strftime("%Y%m%d")
+filename = '{}_{}.zip'.format("smplx_blender", filedate)
+
+root_folder = "{}/../".format(build_script_dir)
+addon_folder = "{}/smplx_blender".format(root_folder)
+data_folder = "{}/data".format(addon_folder)
```

Comment:
Pathlib makes dealing with folders a little nicer. 

```suggestion
from pathlib import Path

build_script_dir = Path(__file__).parent
today = date.today()
filedate = today.strftime("%Y%m%d")
filename = '{}_{}.zip'.format("smplx_blender", filedate)

root_folder = build_script_dir.parent
addon_folder = root_folder / "smplx_blender"
data_folder = addon_folder / "data"
```